.eslintrc.json

```diff
@@ -38,7 +38,9 @@
           "ArrowFunctionExpression": true,
           "FunctionExpression": true
         }
-    }]
+    }],
+    "space-before-blocks":["error", "always"],
```

Comment:
Just add a space after `:`

`"space-before-blocks": ["error", "always"],`

Maybe we can incorporate formatting of this configuration in `npm run lint`. But that is a task for a different PR.

Reply:
> Just add a space after `:`

Okay sure I will update my PR 

> But that is a task for a different PR.

I will open a PR for this too